### ISSUE TITLE
Is there any way to downlaod the comments of a insta post?

### ISSUE THREAD

**Issue by visheshaylani** [neerajsinghsonu/instagram-data-scraper#44]
Hello, I hope you are doing well.
Well, I need to downlaod the comments of a particular public post. Is there any way which can help me to accomplished this task?

**Comment by neerajsinghsonu** [comment]
Hi @visheshaylani,

I am good thank you and hoping same for you.

Downloading comments isn't the part of this script yet and not thinking to improve soon. You can follow the guideline and do some scripting to get or extract the comments.

You can check this post to know more - https://www.diggernaut.com/blog/how-to-scrape-pages-infinite-scroll-extracting-data-from-instagram/

Hope this helps!

Best,
Neeraj Singh

**Comment by visheshaylani** [comment]
Hi @neerajsinghsonu ,
I am fine too.

Neeraj is this script in the working mode?

Actually, I have tried so many scripts, and all results failed. That's why I am asking.


Regards,
Vishesh Kumar

**Comment by neerajsinghsonu** [comment]
Yes, it is working, download the zip and run the code once to check all the features.

**Comment by visheshaylani** [comment]
Bro, 
I am not talking about this repo. I am asking about the article you given me.

**Comment by neerajsinghsonu** [comment]
the blog has idea - how to fetch comments for a post. 

You have to code to get it. And yes the idea is still in practice.

Check - "view-source:https://www.instagram.com/static/bundles/es6/Consumer.js/1f61c35da03c.js"

Find - "2418469a2b4d9b47ae7bec08e3ec53ad"

and then check this - "https://www.instagram.com/graphql/query/?query_hash=2418469a2b4d9b47ae7bec08e3ec53ad&variables=%7B%22shortcode%22%3A%22CEt6D7ODLhs%22%2C%22child_comment_count%22%3A3%2C%22fetch_comment_count%22%3A40%2C%22parent_comment_count%22%3A24%2C%22has_threaded_comments%22%3Atrue%7D"

Instagram use "https://www.instagram.com/graphql/query/?query_hash=" endpoint to fetch all details about a post.

Hope this helps!
